kube/stateful_set.go

```diff
@@ -52,7 +52,7 @@ func NewStatefulSet(role *model.Role, settings *ExportSettings) (*v1beta1.Statef
 }
 
 // getVolumeClaims returns the list of persistent volume claims from a role
-func getVolumeClaims(role *model.Role) []v1.PersistentVolumeClaim {
+func getVolumeClaims(role *model.Role, createHealmChart bool) []v1.PersistentVolumeClaim {
```

Comment:
I agree with the comment and will change it if there is any other change required.